core/src/components/picker-column-internal/picker-column-internal.scss

```diff
@@ -77,6 +77,14 @@
   cursor: default;
 }
 
+:host(.picker-column-disabled) {
+  overflow-y: hidden;
+
+  .picker-item {
```

Comment:
I think we could group this selector with the following styles to have the opacity managed from a single style:
```scss
:host .picker-item.picker-item-disabled,
:host .picker-item.picker-column-disabled {
  opacity: 0.4;
}
```